sdk/Lusid.Sdk.Tests/tutorials/Valuations/InlineBondValuation.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Valuations
+{
+    [TestFixture]
+    public class InLineBondValuation
+    {
+        private ILusidApiFactory _apiFactory;
+
+        // This defines the scope that entities will be created in
+        private const string TutorialScope = "Testdemo";
+
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _apiFactory = LusidApiFactoryBuilder.Build("secrets.json");
+        }
+        
+        [Test]
+        public void Run_BondInLineValuation()
+        {
+            // Define a flow convention for the Bond
+            // In this instance, the bond will pay a GBP coupon every 6 months
+            // Accrued interest will be calculated on the ACT/365 day count convention
+            
+            var flowConventions = new FlowConventions(
+                scope: null,
+                code: null,
+                currency: "GBP",
+                paymentFrequency: "6M",
+                rollConvention: FlowConventions.RollConventionEnum.MF,
+                dayCountConvention: FlowConventions.DayCountConventionEnum.Act365,
+                holidayCalendars: new List<string>(),
+                settleDays: 2,
+                resetDays: 2
+            );
+            
+            // Use the GetValuationOfWeightedInstruments to price our Bond
+            // We price the Bond for a date between the Bond's startDate and maturityDate
+
+            ListAggregationResponse bondValuationResult = _apiFactory.Api<IAggregationApi>().GetValuationOfWeightedInstruments(
+                new InlineValuationRequest(recipeId: new ResourceId(scope: "MyRecipeScope", code: "default"),
+                    valuationSchedule: new ValuationSchedule(
+                        effectiveFrom: new DateTimeOrCutLabel("2020-09-01"),
+                        effectiveAt: new DateTimeOrCutLabel("2020-09-01"),
```

Comment:
effectiveAt = effectiveFrom is redundant, it would default to just that single day.
tenor = "1D" is by default and can be omitted if you wish

Reply:
Thanks @jz-finbourne 
I have now removed `tenor`
Question - what's the correct valuationSchedule config if I just want to value for one day? Using effectiveAt and effectiveFrom seems to give me the correct results?

Reply:
For valuation of one day, supplying (just) effectiveAt should be sufficient. 
ValuationSchedule(effectiveAt: someDate, effectiveFrom: someDate) is equivalent to just ValuationSchdule(effectiveAt: someDate).  Either works although latter is probably preferred as it is cleaner and internally runs less code (so more efficient and less error-prone).

Reply:
Thanks for quick response @jz-finbourne 
Updated in latest commit